### ISSUE TITLE
Failing test - [sig-autoscaling] [Feature:HPA] [Serial] [Slow] Horizontal pod autoscaling (non-default behavior) with long upscale stabilization window should scale up only after the stabilization period

### ISSUE THREAD

**Issue by jsturtevant** [kubernetes/kubernetes#110644]
### Which jobs are failing?

https://testgrid.k8s.io/sig-windows-master-release#capz-windows-containerd-master-serial-slow

### Which tests are failing?

Kubernetes e2e suite.[sig-autoscaling] [Feature:HPA] [Serial] [Slow] Horizontal pod autoscaling (non-default behavior) with long upscale stabilization window should scale up only after the stabilization period

### Since when has it been failing?

6/13

### Testgrid link

https://prow.k8s.io/view/gs/kubernetes-jenkins/logs/ci-kubernetes-e2e-capz-master-containerd-windows-serial-slow/1537585905907273728

### Reason for failure (if possible)

```
Jun 15 02:05:02.737: INFO: RC consumer: sending request to consume 220 millicores
Jun 15 02:05:02.737: INFO: ConsumeCPU URL: {https   capz-conf-bn7x30-61a41b7b.eastus.cloudapp.azure.com:6443 /api/v1/namespaces/horizontal-pod-autoscaling-5573/services/consumer-ctrl/proxy/ConsumeCPU  false durationSec=30&millicores=220&requestSizeMillicores=100  }
Jun 15 02:05:22.606: INFO: waiting for 2 replicas (current: 4)
Jun 15 02:05:32.608: INFO: waiting for 2 replicas (current: 4)
Jun 15 02:05:32.608: INFO: Unexpected error: timeout waiting 3m30s for 2 replicas: 
    <*errors.errorString | 0xc0002362b0>: {
        s: "timed out waiting for the condition",
    }
Jun 15 02:05:32.608: FAIL: timeout waiting 3m30s for 2 replicas: timed out waiting for the condition

Full Stack Trace
k8s.io/kubernetes/test/e2e/autoscaling.glob..func7.2.1()
	test/e2e/autoscaling/horizontal_pod_autoscaling_behavior.go:131 +0x38f
k8s.io/kubernetes/test/e2e.RunE2ETests(0x2567617?)
	test/e2e/e2e.go:130 +0x686
k8s.io/kubernetes/test/e2e.TestE2E(0x24d8d19?)
	test/e2e/e2e_test.go:136 +0x19
testing.tRunner(0xc0008a89c0, 0x73e4400)
	/usr/local/go/src/testing/testing.go:1439 +0x102
created by testing.(*T).Run
	/usr/local/go/src/testing/testing.go:1486 +0x35f
ï¿½[1mSTEPï¿½[0m: Removing consuming RC consumer
```

```
Jun 15 02:05:48.142: INFO: At 2022-06-15 02:01:39 +0000 UTC - event for consumer-56d5494df8-4w992: {kubelet capz-conf-lhj8f} Started: Started container consumer
Jun 15 02:05:48.142: INFO: At 2022-06-15 02:01:39 +0000 UTC - event for consumer-56d5494df8-bgdb5: {kubelet capz-conf-lhj8f} Started: Started container consumer
Jun 15 02:05:48.142: INFO: At 2022-06-15 02:01:40 +0000 UTC - event for consumer-56d5494df8-kvxx7: {kubelet capz-conf-knzdm} Started: Started container consumer
Jun 15 02:05:48.142: INFO: At 2022-06-15 02:01:40 +0000 UTC - event for consumer-56d5494df8-t5p8q: {kubelet capz-conf-knzdm} Started: Started container consumer
Jun 15 02:05:48.142: INFO: At 2022-06-15 02:01:44 +0000 UTC - event for consumer-ctrl: {replication-controller } SuccessfulCreate: Created pod: consumer-ctrl-vx82m
Jun 15 02:05:48.142: INFO: At 2022-06-15 02:01:44 +0000 UTC - event for consumer-ctrl-vx82m: {default-scheduler } Scheduled: Successfully assigned horizontal-pod-autoscaling-5573/consumer-ctrl-vx82m to capz-conf-lhj8f
Jun 15 02:05:48.142: INFO: At 2022-06-15 02:01:46 +0000 UTC - event for consumer-ctrl-vx82m: {kubelet capz-conf-lhj8f} Pulled: Container image "registry.k8s.io/e2e-test-images/agnhost:2.39" already present on machine
Jun 15 02:05:48.142: INFO: At 2022-06-15 02:01:47 +0000 UTC - event for consumer-ctrl-vx82m: {kubelet capz-conf-lhj8f} Created: Created container consumer-ctrl
Jun 15 02:05:48.143: INFO: At 2022-06-15 02:01:48 +0000 UTC - event for consumer-ctrl-vx82m: {kubelet capz-conf-lhj8f} Started: Started container consumer-ctrl
Jun 15 02:05:48.143: INFO: At 2022-06-15 02:05:42 +0000 UTC - event for consumer: {endpoint-controller } FailedToUpdateEndpoint: Failed to update endpoint horizontal-pod-autoscaling-5573/consumer: Operation cannot be fulfilled on endpoints "consumer": the object has been modified; please apply your changes to the latest version and try again
Jun 15 02:05:48.143: INFO: At 2022-06-15 02:05:42 +0000 UTC - event for consumer-56d5494df8-4w992: {kubelet capz-conf-lhj8f} Killing: Stopping container consumer
Jun 15 02:05:48.143: INFO: At 2022-06-15 02:05:42 +0000 UTC - event for consumer-56d5494df8-bgdb5: {kubelet capz-conf-lhj8f} Killing: Stopping container consumer
Jun 15 02:05:48.143: INFO: At 2022-06-15 02:05:42 +0000 UTC - event for consumer-56d5494df8-kvxx7: {kubelet capz-conf-knzdm} Killing: Stopping container consumer
Jun 15 02:05:48.143: INFO: At 2022-06-15 02:05:42 +0000 UTC - event for consumer-56d5494df8-t5p8q: {kubelet capz-conf-knzdm} Killing: Stopping container consumer
Jun 15 02:05:48.143: INFO: At 2022-06-15 02:05:45 +0000 UTC - event for consumer-ctrl-vx82m: {kubelet capz-conf-lhj8f} Killing: Stopping container consumer-ctrl
```

### Anything else we need to know?

looks like this was a new test added in https://github.com/kubernetes/kubernetes/commit/a724818ad0430cabe117c5e64afdaecfb8dd51c8

### Relevant SIG(s)

/sig autoscaling
/sig windows

**Comment by jsturtevant** [comment]
diff when it started failing https://github.com/kubernetes/kubernetes/compare/c0cc9116677732...10066243df4313

**Comment by Shubham82** [comment]
Hi @jsturtevant 
This issue is a Duplicate of #110662 and the PR for this issue is here: #110664

**Comment by jsturtevant** [comment]
/close
in preference of #110662 and revert in #110664
